pkgs/by-name/va/valkey/package.nix

```diff
@@ -32,7 +33,13 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-IzerctJUc478dJu2AH20s/A3psiAZWDjQG3USQWqpos=";
   };
 
-  patches = lib.optional useSystemJemalloc ./use_system_jemalloc.patch;
+  patches = [
+    (fetchpatch2 {
```

Comment:
Using fetchpatch2 requires adding `?full_index=1` to the patch URL.

Reply:
This bug in fetchpatch2: #257446

The test (for fetchpatch2) is a bit broken and was fixed at least once already: #323824
I'll update the documentation to warn about this issue.